tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -114,21 +41,19 @@ def organization_ak_setup(module_org):
     subscription = entities.Subscription(organization=module_org).search(
         query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
     )[0]
-    ak.add_subscriptions(data={"quantity": 10, "subscription_id": subscription.id})
+    ak.add_subscriptions(data={'quantity': 10, 'subscription_id': subscription.id})
     return module_org, ak
 
 
-@pytest.mark.libvirt_content_host
 @pytest.fixture(scope="module")
 def virtual_host():
-    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as vm:
         yield vm
 
 
-@pytest.mark.libvirt_content_host
 @pytest.fixture(scope="module")
 def baremetal_host():
-    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as vm:
```

Comment:
you can use the predefined rhel7_contenthost fixture for these

Reply:
@JacobCallahan I can't use `rhel7_contenthost` fixture twice in a test. I need two hosts here, and I also need to set `set_infrastructure_type` for one host, that's why there are two fixtures(`baremetal_host` and `virtual_host`). 
There's `_count` option in `VMBroker` to get two vm's but current way looks better. what you think?

Reply:
when you need both hosts, you can definitely use the context manager with `_count` because it checks out multiple hosts at the same time, instead of sequentially via fixtures.